Aztec/Classes/TextKit/TextView.swift

```diff
@@ -488,7 +488,18 @@ open class TextView: UITextView {
         forceRedrawCursorAfterDelay()
     }
 
-
+    private lazy var paragraphFormatters: [AttributeFormatter] = [
```

Comment:
I just tried changing this to `private let paragraphFormatters` and it seems to work fine.

Thoughts about switching it?